src/Umbraco.Core/Notifications/UmbracoApplicationStartedNotification.cs

```diff
@@ -5,5 +5,26 @@ namespace Umbraco.Cms.Core.Notifications
     /// </summary>
     /// <seealso cref="Umbraco.Cms.Core.Notifications.INotification" />
     public class UmbracoApplicationStartedNotification : INotification
-    { }
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UmbracoApplicationStartedNotification" /> class.
+        /// </summary>
+        public UmbracoApplicationStartedNotification()
+            : this(false)
+        { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UmbracoApplicationStartedNotification" /> class.
+        /// </summary>
+        /// <param name="isRestarting">Indicates whether Umbraco is restarting.</param>
+        public UmbracoApplicationStartedNotification(bool isRestarting) => IsRestarting = isRestarting;
```

Comment:
Because this notification is added in the (currently) unreleased 9.3.0 version, the constructor could also be updated to:
```suggestion
        /// <summary>
        /// Initializes a new instance of the <see cref="UmbracoApplicationStartedNotification" /> class.
        /// </summary>
        /// <param name="isRestarting">Indicates whether Umbraco is restarting.</param>
        public UmbracoApplicationStartedNotification(bool isRestarting = false) => IsRestarting = isRestarting;
```